system-variables.md

```diff
@@ -1451,10 +1444,10 @@ Usage example:
 SET tidb_query_log_max_len = 20
 ```
 
-### `tidb_read_staleness` <span class="version-mark">New in v5.4.0</span>
+### tidb_read_staleness <span class="version-mark">New in v5.4.0</span>
 
 - Scope: SESSION
-- Default value: `0`
+- Default value: ""
```

Comment:
According the the source, the default is "".  But it might be a bug: https://github.com/pingcap/tidb/issues/32763

In either case, it's correct right now - but it should be changed in future to be numeric. This will also help improve the docs as the type is shown.